app/forms/collection_form.rb

```diff
@@ -1,30 +1,36 @@
 # frozen_string_literal: true
 
 # Form for a Collection
-class CollectionForm < ApplicationForm
+class CollectionForm < RefactoredApplicationForm
   accepts_nested_attributes_for :related_links, :contact_emails, :managers, :depositors, :reviewers, :contributors
+
   before_validation do
-    blank_managers = managers_attributes.select(&:empty?)
```

Comment:
The prior implementation supported readers for nested attributes both with and without the `_attributes` suffix, which was a departure from ActiveRecord. Now the `..._attributes` form is only supported for writes (the `#..._attributes=` form instance method).